core/trino-main/src/test/java/io/trino/sql/gen/BenchmarkInCodeGenerator.java

```diff
@@ -83,31 +87,43 @@
         public void setup()
         {
             Random random = new Random();
```

Comment:
Random should have predefined seed

Reply:
You noticed that I did not change this code at all (modulo moving to inner class and adding missing parameter on call) ? :)

Reply:
I will change that. But I saw consistent results between fork runs. So I do not expect it will change anything.